ui/components/app/wallet-overview/coin-buttons.stories.js

```diff
@@ -24,7 +24,6 @@ export default {
       string: '0.0031',
     },
     classPrefix: 'coin',
-    iconButtonClassName: '',
```

Comment:
Removed unused prop from icon button and all instances

<img width="322" alt="Screenshot 2025-05-16 at 8 22 53 AM" src="https://github.com/user-attachments/assets/6bd91a8a-0a52-4b82-9f8b-db8931f0fa8b" />